Plots/Satellites/NCL_sat_1.py

```diff
@@ -0,0 +1,109 @@
+"""
+NCL_sat_1.py
+===============
+This script illustrates the following concepts:
+   - Using 'astype' to unpack 'short' data
+   - Drawing line contours over a satellite map
+   - Changing the view of a satellite map
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/sat_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/sat_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import xarray as xr
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+
+import geocat.datafiles as gdf
+import geocat.viz.util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/slp.1963.nc"), decode_times=False)
+
+# Get lon/lat data from the 24th timestep
+U = ds.slp[24, :, :]
+
+# Translate short values to float values
+U = U.astype('float64')
+
+# Convert Pa to hPa data
+U = U*0.01
+
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
+wrap_U = gvutil.xr_add_cyclic_longitudes(U, "lon")
+
+###############################################################################
+# Create plot
+
+# Set figure size
+fig = plt.figure(figsize=(8, 8))
+
+# Set global axes with an orthographic projection
+proj = ccrs.Orthographic(central_longitude=270, central_latitude=45)
+ax = plt.axes(projection=proj)
+ax.set_global()
```

Comment:
The NCL plot looks slightly zoomed in. Maybe using `ax.set_extent()` could make your plot look the same.

Reply:
Ah okay. I'm not sure how to fix that then.